subserver_permissions.go

```diff
@@ -55,13 +55,28 @@ func getAllMethodPermissions() map[string][]bakery.Op {
 	return result
 }
 
+// denyList is the list of permission actions that will be denied for readonly
+// macaroons.
+// TODO(elle): the individual daemon libraries should export a readonly
```

Comment:
Which can be done mainly by exporting the perms in the various `*rpc` packages?

Reply:
Alternatively, this can function as a white list: iirc they all have "read" in there as the perms right?

Reply:
> Which can be done mainly by exporting the perms in the various *rpc packages?

yes

> Alternatively, this can function as a white list: iirc they all have "read" in there as the perms right?

good idea 👍

Reply:
I think this might cause an issue with Loop since some calls require a `loop:in` or `loop:out` permission in addition to the `terms:read` one (for example `LoopOutTerms`). But we can adjust that later if we find that we need loop terms in read-only mode.